BPKCarouselCardSnapshotTests.swift

```diff
@@ -0,0 +1,77 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2024 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+import Backpack
+
+import SnapshotTesting
+
+class BPKCarouselSnapshotTest: XCTestCase {
```

Comment:
I believe this file should be in the `SnapshotTests` folder.

Reply:
Hmmm...I am missing something here 😅 Why do we need this file? 🤔

Reply:
Removed as cannot remember why added